content/docs/faq.md

```diff
@@ -224,7 +224,7 @@ By itself, Scratch Addons is just an addon loader. Its main tasks are:
 
 ### How can I add/remove myself from the contributors page?
 
-If you want your name to be on the page, please read and follow the instructions of [this issue](https://github.com/ScratchAddons/contributors/issues/{{< specifics/contributors-issue >}}).
+If you want your name to be on the page, please read and follow the instructions of [this issue](https://github.com/ScratchAddons/contributors/issues/183).
```

Comment:
Please leave this alone. The way it works is that it takes the number from `config.yml`, located on the root.

Reply:
@Hans5958 , this is what it loads to be 😕...
![image](https://user-images.githubusercontent.com/82447031/181241556-f24bd68a-14e8-4da3-83ac-5093f5582d8f.png)

Reply:
That's because it is something that Hugo would convert. You need to use it if you want to see the real action, but just trust me that Hugo will take care of it.

Reply:
alright then 👍